packages/core/src/types.ts

```diff
@@ -1186,7 +1202,11 @@ export type Mapper<
   TContext extends MachineContext,
   TEvent extends EventObject,
   TParams extends {}
-> = (args: { context: TContext; event: TEvent }) => TParams;
+> = (args: {
+  context: TContext;
+  event: TEvent;
+  self: ActorRef<TEvent>;
```

Comment:
this has been added here without any test coverage 😢

Reply:
Sorry! I'll add tests